apps/src/Observer.js

```diff
@@ -0,0 +1,52 @@
+/** @file Helper for consuming ObservableEvents. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
```

Comment:
I'll go to 120 for all the files with strict linting.

I'd like to do linting changes in another PR since it'll touch a lot of non
- *Lab files. Cool?
On Feb 3, 2016 2:56 PM, "Brent Van Minnen" <notifications@github.com> wrote:

> In apps/src/Observer.js
> <https://github.com/code-dot-org/code-dot-org/pull/6626#discussion_r51802317>
> :
>
> > @@ -0,0 +1,52 @@
> > +/** @file Helper for consuming ObservableEvents. */
> > +/* jshint
> > + funcscope: true,
> > + newcap: true,
> > + nonew: true,
> > + shadow: false,
> > + unused: true,
> > + eqeqeq: true,
> > +
> > + maxlen: 90,
> > + maxparams: 3,
> > + maxstatements: 200
>
> Cool. Generally fine with your decisions. Seems a little weird to
> introduce 90 as maxlen, when our style guide already waffles between 80 and
> 120. Why not just set maxlen to one of those?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6626/files#r51802317>.
>